### DISCUSSION TITLE
Compiling SuperSlicer give a lot of warnings

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#2746] [Community contribution]
I got severals warning when compiling (a lot about cmake version), compilation is fine but is that normal ? is it planned to remove those warnings ?

![image](https://user-images.githubusercontent.com/12198339/164972094-ac32a950-5b13-4b27-9a9a-29536f763c72.png)

(I am using my own fork and could probably fix it, but would be better if it is done in the original upstream)

**Comment by supermerill** [comment]
will be fixed when merged with 2.5 prusaslicer.

**Reply by DzzD** [reply]
Nice
